matching-service/controller/socketHandler/pendingMatchHandler.js

```diff
@@ -18,7 +18,8 @@ const pendingMatchHandler = (io) => {
                 let question; 
 
                 //retrive easy question by sending a GET API to question-service
-                axios.get('http://localhost:8002/api/questions/?level=easy')
+                //NOTE: use this url instead if running locally without docker: 'http://localhost:8002/api/questions/?level=easy'
+                axios.get('http://question-service:8002/api/questions/?level=easy')
```

Comment:
Perhaps can define the base url for question-service as a constant somewhere (in a separate file or line of code), so that it is easier to make changes

Reply:
Was redefined in `config.js`